save-backend/src/main/kotlin/com/saveourtool/save/backend/service/ProjectService.kt

```diff
@@ -51,6 +48,52 @@ class ProjectService(
         return Pair(projectId, projectSaveStatus)
     }
 
+    /**
+     * Mark organization with [project] as deleted
+     *
+     * @param newStatus is new status for [project]
+     * @param project is organization in which the status will be changed
+     * @return project
+     */
+    @Suppress("UnsafeCallOnNullableType")
+    private fun changeProjectStatus(project: Project, newStatus: ProjectStatus): Project = project
+        .apply {
+            status = newStatus
+        }
+        .let {
+            projectRepository.save(it)
+        }
+
+    /**
+     * Mark organization [project] as deleted
+     *
+     * @param project an [project] to delete
+     * @return deleted organization
+     */
+    fun deleteProject(project: Project): Project =
```

Comment:
`@Transactional`

Reply:
I looked, they don't seem to change the values of several tables at once, then I can delete in this file. Or is it better to do it everywhere?

Reply:
ok